common/src/sql/dbinit.sql

```diff
@@ -72,6 +74,35 @@ CREATE TABLE omicron.public.sled (
     last_used_address INET NOT NULL
 );
 
+/*
+ * Services
```

Comment:
In this PR, this table isn't actually being *read*, but it can be *written*.

I expect this will be the future format Nexus uses to identify "what services are running where, and where should I boot new services".